{"id": 3884, "title": "Ticket #3884: possible critical 8.0.2 dagman recovery bug", "description": "<blockquote>\nPeter writes:\n\n<p></p><pre>  On recovery, DAGMan appears not to recognize already-completed jobs it has\n  logged as completed, causing it to freak out when it sees their children.\n  The log below spells it out clearly.\n</pre>\n\n<p></p><pre>  We're testing now to confirm that this /doesn't/ happen in 8.0.1, but we\n  know it does happen in 8.0.2.</pre>\n</blockquote>", "remarks": "<blockquote>\nDup of <span class=\"ticket\"><a class=\"defer\" href=\"/tickets?ticket=3882\" onclick=\"get_ticket_and_populate_wrapper('3882'); return false;\" title=\"DAGMan recovery aborts after failing to recognize completed jobs\">#3882</a></span>.</blockquote>", "derived_tickets": "", "attachments": "", "check_ins": "", "type": "incident", "last_change": "2013-Aug-29 16:01", "status": "defer", "created": "2013-Aug-29 14:48", "fixed_version": "2013-Aug-29 14:48", "broken_version": "v080002", "priority": "5", "subsystem": "Dag", "assigned_to": "", "derived_from": "", "creator": "tlmiller", "rust": "s8336", "customer_group": "ligo", "visibility": "public", "notify": "pfcouvar@syr.edu, tlmiller@cs.wisc.edu,pcouvare@caltech.edu", "due_date": ""}